lib/screen/interactive_postcard/prompt_page.dart

```diff
@@ -56,6 +60,14 @@ class _PromptPageState extends State<PromptPage> {
                       child: TextField(
                         controller: _controller,
                         textAlign: TextAlign.center,
+                        onChanged: (value) {
+                          final isValid = _isValidPrompt(value);
+                          if (isValid != _enableSave) {
+                            setState(() {
+                              _enableSave = isValid;
```

Comment:
please use another name instead of `_enableSave`, because _enableSave will make confuse

Reply:
what do you suggest

Reply:
bool _isPromptValid or _isValid